pipeline/mutate/mutator_hydrator.go

```diff
@@ -78,21 +84,53 @@ type externalAPIConfig struct {
 }
 
 type MutatorHydratorConfig struct {
-	Api externalAPIConfig `json:"api"`
+	Api      externalAPIConfig `json:"api"`
+	CacheTtl string            `json:"cache_ttl"`
 }
 
 type mutatorHydratorDependencies interface {
 	x.RegistryLogger
 }
 
 func NewMutatorHydrator(c configuration.Provider, d mutatorHydratorDependencies) *MutatorHydrator {
-	return &MutatorHydrator{c: c, d: d, client: httpx.NewResilientClientLatencyToleranceSmall(nil)}
+	cache, _ := ristretto.NewCache(&ristretto.Config{
+		NumCounters: 10000,
+		MaxCost:     1 << 25,
```

Comment:
How much RAM will this use at maximum use? Should this be configurable?

Reply:
I think it should be configurable, not sure the best way to do that as the cache is created when it is initialized, but the config comes through on the hydrate method, any suggestions?

Reply:
Yeah I think that makes sense but we can also just leave it as is. I think we can add config later if it's really required!